usaspending_api/references/tests/test_tas_autocomplete.py

```diff
@@ -0,0 +1,145 @@
+import json
+
+from django.test import TestCase
+from model_mommy import mommy
+
+
+BASE_ENDPOINT = "/api/v2/autocomplete/accounts/"
+
+
+class TASAutocompleteTestCase(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make("references.CGAC", cgac_code="000", agency_name="Agency 000", agency_abbreviation="A000")
+        mommy.make("references.CGAC", cgac_code="002", agency_name="Agency 002", agency_abbreviation="A002")
```

Comment:
Could we have a test case to make sure a string that doesn't start but ends with the search text isn't found?

Reply:
👍